packages/workspace/src/browser/workspace-delete-handler.ts

```diff
@@ -37,6 +37,10 @@ export class WorkspaceDeleteHandler implements UriCommandHandler<URI[]> {
         return !!uris.length && !this.isRootDeleted(uris);
     }
 
+    isEnabled(uris: URI[]): boolean {
+        return !!uris.length && !this.isRootDeleted(uris);
+    }
+
     protected isRootDeleted(uris: URI[]): boolean {
```

Comment:
This line is not part of this change but the function should be renamed imo. 

What it really means is "can this folder be deleted". i suggested to renamed it into "canDelete" or "isRoot". what do you think?

Reply:
I agree, `isRootDeleted` should probably be rename to something more meaningful. When making the change I was confused on what it actually achieved.